docs/build.md

```diff
@@ -0,0 +1,61 @@
+# Installation
+
+## Installation using Spack
+
+The simplest way to install ArborX is to use [Spack](https://github.com/spack/spack) package manager:
+```shell
+spack install arborx
+```
+
+## Manual installation
+
+ArborX has the following dependencies:
+
+Dependency | Version | Required
+--- | --- | ---
+Kokkos | 2.8.00 | :heavy_check_mark:
+MPI    | 2      |
+CMake  | 3.14   | :heavy_check_mark:
+
+The dependencies may be installed using [Spack](https://github.com/spack/spack) package manager:
+FIXME (try out Kokkos install)
+```shell
+spack install kokkos
+spack install mpi
+```
+or manually (cmake and mpi - easy, Kokkos is here):
+FIXME: do we need instructions to install Kokkos here?
```

Comment:
I would link to `https://github.com/kokkos/kokkos/blob/develop/BUILD.md` or similar.